src/utils/url.js

```diff
@@ -20,14 +21,15 @@ const defaultUrl = {
 };
 
 export function parse(url: string): URL | object {
+  const strippedUrl = unformatUrl(url);
```

Comment:
probably we want to take this out of parse, if we want to preserve " [sm]" in the text.

Reply:
Yep. Perhaps. We could also format it ourselves in the render function. [sm] is kinda ugly :) we just need to know it's original

Reply:
or we just don't render ' [sm]' as it is right now?

Reply:
sure